lib/htmlsanitycheckrc.json

```diff
@@ -1,9 +1,17 @@
 {
   "sourceDir": ".",
-  "extension": ["html"],
+  "extension": [
+    "html"
+  ],
   "recursive": true,
   "package": "./package.json",
-  "reporter": "junit",
+  "reporter": {
+    "jUnitXmlReporterConfig": {
+      "importPath": "./reporter/JUnitXmlReporter",
```

Comment:
why is it necessary the importPath to configure, is it not only the same path and then based on key a specific class?

Reply:
mapping is now in utils, as requested